parts/kubernetesmasteraddons-tiller-deployment.yaml

```diff
@@ -5,13 +5,15 @@ metadata:
   namespace: kube-system
   labels:
     kubernetes.io/cluster-service: "true"
```

Comment:
@jackfrancis @lachie83  If they're both present, `addonmanager.kubernetes.io/mode=EnsureExists` will take precedence. If only `kubernetes.io/cluster-service: "true"` is present then addon-manager will treat it as `addonmanager.kubernetes.io/mode=Reconcile`. 

Lachie suggested removing the deprecation in another PR which is OK by me. That said, I think it's a lot clearer to remove the deprecated label and use the actual, used value for new cluster.